src/AgGrid/ValueFormat.elm

```diff
@@ -167,6 +168,25 @@ numberFilterValueGetter field =
     """ [ field ]
 
 
+{-| Format a cell value as Date.
+
+This is similar to using a normal Value Getter, but is specific to the filter.
+
+    cellValue = "2023-12-12"
+
+    dateFilterValueGetter "dateField"
+    > new Date("2023-12-12")
+
+-}
+dateFilterValueGetter : String -> String
+dateFilterValueGetter field =
+    String.Interpolate.interpolate """
+        if (!data) { return null; }
+
+        return new Date(data.birthday)
```

Comment:
The example didn't work for me when I tried checking for equality. But greatherThan/lessThan worked fine. I suspect that the time might be a bit off due to timezones.

Reply:
I expected AgGrid calls `new Date()` just as our ValueFormatter does (thus both being the same time zone), but I did not expect this:

![screenshot-2023-12-13T14:07:54+01:00](https://github.com/mercurymedia/elm-ag-grid/assets/693447/cbe540bd-ce5e-4233-ab5a-835081d80390)